tests/tests/cpld_bmark_wave_v16_p7b_35d

```diff
@@ -0,0 +1,163 @@
+#
```

Comment:
We'll need to add this 35d test to the rt_35d.conf along w/ adding the suite to the compile line there. But you may still be making changes/testing things?

Reply:
@DeniseWorthen the 35d test is tested and added to rt_35d.conf. Please review the PR.